public/html/index.html

```diff
@@ -32,7 +32,9 @@
     />
   </head>
   <body>
+    <div id="splash-screen"></div>
```

Comment:
Does it need to be a new div element, vs just a class on the main div?

Reply:
adding new div is more extendable, e.g. we can add loader there

Reply:
I think we can do the same thing with not new div or class and just using the existing `light` and `dark` class we already have

Reply:
it seems `light/dark` classes are used by theme provider. 
if we use it, we need to know which theme user selected: `light/dark/system` at the start of app, otherwise it will lead to bugs

Reply:
currently splash screen relies just on system theme